### ISSUE TITLE
Syntax Error

### ISSUE THREAD

**Issue by caderyn19** [juliancwirko/nft-art-maker#27]
I'm following the tutorial video you posted on YT, and I'm getting a syntax error. I'm not quite sure what I'm doing wrong:
```
file:///usr/local/lib/node_modules/nft-art-maker/node_modules/@web-std/fetch/src/index.js:110
			response_?.emit("error", err);
			          ^

SyntaxError: Unexpected token '.'
    at Loader.moduleStrategy (internal/modules/esm/translators.js:140:18)
```

If I try the minimum recommended node, 14.14.0, or higher it gives me:
```
Error: The module '/usr/local/lib/node_modules/nft-art-maker/node_modules/canvas/build/Release/canvas.node'
was compiled against a different Node.js version using
NODE_MODULE_VERSION 72. This version of Node.js requires
NODE_MODULE_VERSION 93. Please try re-compiling or re-installing
the module (for instance, using `npm rebuild` or `npm install`).
```

If I downgrade to version 72, it throws the SyntaxError...
I'm running Ubuntu 22.04.1 LTS.

**Comment by juliancwirko** [comment]
You may need to reinstall it:
```
npm remove nft-art-maker -g
```
```
npm install nft-art-maker -g
```

Make sure that you use proper version of Node: `node -v` to  check

on which version of Node would you like to run it?

**Comment by caderyn19** [comment]
Ahhh that helped! Now I got to configuring layers, so I'm back on track.
Thank you!

I uninstalled npm, did a clean install to make sure it was fresh, used nvm
to install 14.14.0 (was on 12.22.12 apparently, but that was the
"NODE_MODULE_VERSION 72" it was throwing at me), then did the removal and
install of nft-art-maker and it brought me to the next step.

Thank you for that simple solution! I thought I did that, but I did rebuild
instead of remove and install. :facepalm:
